cli/Cargo.toml

```diff
@@ -10,15 +10,15 @@ reqwest = { version = "0.11", features = ["blocking", "json"] }
 base64 = "0.13"
 log = "0.4"
 pretty_env_logger = "0.4"
-termion = "1.5"
-tls_codec = "0.2.0"
+tls_codec = { version = "0.2.0", features = ["derive", "serde_serialize"] }
 
 openmls = { path = "../openmls", features = ["test-utils"] }
 ds-lib = { path = "../delivery-service/ds-lib" }
 openmls_traits = { path = "../traits" }
 openmls_rust_crypto = { path = "../openmls_rust_crypto" }
 openmls_memory_keystore = { path = "../memory_keystore" }
 
-# Patching unreleased crates
-[patch.crates-io]
-tls_codec = { git = "https://github.com/RustCrypto/formats.git", features = ["derive", "serde_serialize"] }
+[dependencies.termion]
+version = "1.5"
+git = "https://gitlab.redox-os.org/Jezza/termion.git"
```

Comment:
The cli wasn't compiling anymore on windows because of termion. This requires an unreleased branch of termion. That's what this change is about. Without this, the project wouldn't simply compile on windows